### PR TITLE
add libiodbc formula, based on work by eugenejen.

### PR DESCRIPTION
**Pull request by maparent**
Good day! This is based on an earlier pull request by eugenejen: http://github.com/Homebrew/homebrew/pull/20792
I have updated to the latest iodbc version.

### CODE REVIEW

File: Library/Formula/libiodbc.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+require 'formula'
```
Comment (bfontaine) [inline Library/Formula/libiodbc.rb]: Please remove this line.

File: Library/Formula/libiodbc.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+class Libiodbc < Formula
+  homepage 'http://www.iodbc.org/dataspace/iodbc/wiki/iODBC/'
```
Comment (bfontaine) [inline Library/Formula/libiodbc.rb]: Use double-quoted strings everywhere.

File: Library/Formula/libiodbc.rb [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  url 'http://sourceforge.net/projects/iodbc/files/iodbc/3.52.10/libiodbc-3.52.10.tar.gz/download'
+  sha1 '2a8328cea84a3ddb78b10a009652cdd3f1d39fb9'
```
Comment (bfontaine) [inline Library/Formula/libiodbc.rb]: Use a `sha256` here.

File: Library/Formula/libiodbc.rb [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+  depends_on 'automake' => :build
+  depends_on 'autoconf' => :build
+  depends_on 'libtool' => :build
+
+  def install
+    # run bootstrap.sh
```
Comment (bfontaine) [inline Library/Formula/libiodbc.rb]: You can remove this comment.

File: Library/Formula/libiodbc.rb (hunk 2 above)
Comment (bfontaine) [inline Library/Formula/libiodbc.rb]: Also, could you add a short description above the homepage using `desc "…"`? Thanks!

**Comment by maparent** [comment]
Thanks a lot for the comments; I think they're all addressed now.

**Comment by DomT4** [comment]
Few nits left:

```
==> brew style libiodbc

1 file inspected, no offenses detected
Error: 3 problems in 1 formula

==> audit problems
libiodbc:
 * Stable: Don't use /download in SourceForge urls (url is http://sourceforge.net/projects/iodbc/files/iodbc/3.52.10/libiodbc-3.52.10.tar.gz/download).
 * Stable: Use https://downloads.sourceforge.net to get geolocation (url is http://sourceforge.net/projects/iodbc/files/iodbc/3.52.10/libiodbc-3.52.10.tar.gz/download).
 * Description is too long. "name: desc" should be less than 80 characters (currently 97).
```

**Comment by maparent** [comment]
The fail mentions that these files shadow system header files. Those header files disappeared in OSX 10.9, because Apple deprecated iodbc from that version. Is the test system earlier than 10.9?

**Comment by DomT4** [comment]
We test the most recent 3 stable major OS X Versions, so for now 10.8, 10.9 and 10.10. Maybe worth adding something like `keg_only :provided_pre_mountain_lion` so this doesn't cause problems on older systems, although that won't work for ML itself.

**Comment by maparent** [comment]
added the keg_only tag, as requested. Still fails on ML of course.

**Comment by DomT4** [comment]
We can ignore the ML failure if necessary. Going to ask @mistydemeo to comment on whether kegging this is appropriate/useful for <10.8 - Misty runs Tigerbrew and has better experience of not breaking older OS X versions than I do :wink:.

**Comment by maparent** [comment]
Fair. As I said, the newer (3.52.10) include files would not break linking against the older version of iodbc used by OS X (3.52.6).

**Comment by DomT4** [comment]
@BrewTestBot test this please

**Comment by maparent** [comment]
Hello! I would like to know what happened to this merge? I thought absence of test results would not affect the merge. Or is there some more action required on my part?
Thank you,
Marc-Antoine

**Comment by DomT4** [comment]
@BrewTestBot test this please

**Comment by DomT4** [comment]
@mikemcquaid How much do we care about this clashing with 10.8 system headers? We made it `keg_only` on 10.7 and below, but we don't have a `keg_only :provided_pre_mavericks` feature (Although it wouldn't be difficult to add one I guess).

**Comment by mikemcquaid** [comment]
> keg_only :provided_pre_mavericks feature (Although it wouldn't be difficult to add one I guess).

Yeh, would you mind adding that?

**Comment by mikemcquaid** [comment]
At very least I'd like this not to fail the audit.

**Comment by DomT4** [comment]
> Yeh, would you mind adding that?

Sure :+1:

**Comment by DomT4** [comment]
@mikemcquaid #42460

**Comment by DomT4** [comment]
If you update your Homebrew, then rebase this branch against master, you should be able to use `keg_only provided_pre_mavericks` now.

**Comment by maparent** [comment]
Done, thank you!

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format for new formulae is `libiodbc 3.52.10 (new formula)`.

**Pull request closed without merging**